### PR TITLE
Add achandr2 beaverton file

### LINKED ISSUE #7: Create a new file for achandr2_BEAVERTON
- [x] Create a branch 
- [x] Add Commits 
- [x] Open a PR 
- [ ] have a discussion 
- [ ] Seek Approval 
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by achandr2**
closes #7

### CODE REVIEW

File: achandr2-beaverton.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Beverton, OR 
+### Food Recommendations 
+- City tacos 
+- Rustic Roots 
+- Sublime
```
Comment (rup1nder) [inline achandr2-beaverton.md]: is this the text editor?

**Comment by rup1nder** [comment]
:dollar: :dollar: :world_map:

**Review by rup1nder — APPROVED** [review]
Looks good

File: achandr2-beaverton.md (hunk 1 above)
Comment (achandr2) [reply achandr2-beaverton.md]: Note Pad, saved the file as .CMD

**Review by achandr2 — COMMENTED** [review]
Approved

**Pull request merged**